portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,6 +48,10 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    // Define a data store object.
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
General question, is there an advantage of creating the datastoreservice on every doPost, or can you just create it once and re-use the same object?

Reply:
In the example they provided, the object was created in the doPost function, so I followed that format. I just moved it outside the function to the class!